mora/service/address.py

```diff
@@ -51,24 +51,31 @@
                             url_prefix='/service')
 
 
-def get_relation_for(classobj, value):
-    scope = classobj['scope']
-
-    if not isinstance(value, str):
-        value = str(value)
+def get_relation_for(classobj, addrobj, fallback=None):
```

Comment:
Quasi-unrelated to the PR, but `s/classobj/addrtypeobj/`? Just to help make this function more easily parsable the next time I look at it.